instrumentation/kotlin-coroutines/kotlin-coroutines.gradle

```diff
@@ -0,0 +1,25 @@
+ext {
+  // TODO (trask) currently this is only needed for LambdaGen
+  //  revisit if we don't end up dropping Java 7
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+apply from: "$rootDir/gradle/test-with-kotlin.gradle"
+muzzle {
+  pass {
+    group = 'org.jetbrains.kotlinx'
+    module = 'kotlinx-coroutines-core'
+    versions = "[1.0.0,1.3.8)"
```

Comment:
1.4 is not supported?

Reply:
I can't get muzzle to work properly with variants:
```
Execution failed for task ':instrumentation:kotlin-coroutines:muzzle-AssertPass-org.jetbrains.kotlinx-kotlinx-coroutines-core-1.3.9'.
> Could not resolve all files for configuration ':instrumentation:kotlin-coroutines:muzzle-AssertPass-org.jetbrains.kotlinx-kotlinx-coroutines-core-1.3.9'.
   > Could not resolve org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9.
     Required by:
         project :instrumentation:kotlin-coroutines
      > Cannot choose between the following variants of org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9:
```
I've tried diving into the muzzle plugin and playing with setting various properties on the `DefaultArtifact` constructor to have it select the right one, but nothing has worked so far...